pages/docs/installation/installation-bindings-julia.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Julia bindings
+permalink: installation-bindings-julia.html
+keywords: todo, basics, overview, installation, bindings, Julia
+summary: "Use add PreCICE to install the Julia language bindings from PyPI"
+---
+
+## The versioning scheme
+
+The versioning scheme of the Julia bindings is the major and minor preCICE version followed by the version of the julia bindings.
+
+Example: version `1` of the Julia bindings for preCICE version `2.4.0` is `2.4.1`
+
+## Installation
+
+The Julia bindings can be installed with [Pkg.jl](https://pkgdocs.julialang.org/v1/) from the [Julia General Repository](https://github.com/JuliaRegistries/General). For this, preCICE needs to be installed first. For the preCICE installation refer to the [Quickstart](quickstart.html) or the [installation overview](installation-overview.html).
+
+### Adding the package
+
+Add the Julia bindings in the Julia prompt by typing:
+
+```julia-repl
+julia> ]
+pkg> add PreCICE 
+```
+
+Alternatively you can also include the package in a Julia script in the following way:
+
+```julia
+import Pkg; Pkg.add("PreCICE")
+```
+
+### Adding the package from a local folder
+
+If you have cloned or downloaded the Julia bindings on your local machine, add the Julia bindings to your Julia environment in the following way:
+
+```julia-repl
+julia> ]
+pkg> add <path-to-repository>
+```
+
+### Adding a feature branch
+
+If you want to test a new feature, you can install a specific branch of the [PreCICE.jl repository](https://github.com/precice/PreCICE.jl/) with the following command:
+
+```julia-repl
+julia> ]
+pkg> add https://github.com/precice/PreCICE.jl#<branch-name>
+```
+
+## Troubleshooting
+
+If preCICE is installed at a custom path, errors of the form ```ERROR: could not load library "/..."``` can occur after adding the Julia bindings package. Make sure the preCICE library is in the system library path through `echo $LD_LIBRARY_PATH`. If the path containing `libprecice.so` doesn't appear, update the variable with the correct path.
+
+```bash
+~$ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path-to-libprecice.so>
```

Comment:
We usually don't add the `~$` part, but I understand that here this is nice: it clearly shows what is a Bash command and what is a Julia command. Nice!

However, please use braces around variables and double quotes around anything that may contain spaces. Furthermore, whenever you edit such paths, it makes sense to have the most specific path first, as this is the order that the shell looks them up:

```suggestion
~$ export LD_LIBRARY_PATH="<path-to-libprecice.so>:${LD_LIBRARY_PATH}"
```

Btw, you may enjoy exploring https://github.com/koalaman/shellcheck